CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Twilio Verify Android
+## What are good ways to contribute?
+We welcome any kind of contributions. From filling issues, to fixing typos and bugs all the way to creating entirely new templates.
```

Comment:
please change `to creating entirely new templates` to `to implementing improvements`

Reply:
Done